### PR TITLE
Dubbo- #1684

### PR DESCRIPTION
**Pull request by stoneapple**
## What is the purpose of the change
add dubbo-container unit test
XXXXX

## Brief changelog
add dubbo-container unit test

## Verifying this change

add dubbo-container unit test

]

### CODE REVIEW

File: dubbo-container/dubbo-container-spring/src/test/java/org/apache/dubbo/container/spring/MainLauncherTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package org.apache.dubbo.container.spring;
```
Comment (diecui1202) [inline dubbo-container/dubbo-container-spring/src/test/java/org/apache/dubbo/container/spring/MainLauncherTest.java]: Without LICENSE info

File: dubbo-container/dubbo-container-log4j/src/test/java/org/apache/dubbo/container/log4j/Log4jContainerTest.java [hunk 2]
```diff
@@ -17,19 +17,29 @@
...
 /**
  * StandaloneContainerTest
  */
 public class Log4jContainerTest {
+	
+	static final Logger logger = LoggerFactory.getLogger(Log4jContainerTest.class);
+
 
     @Test
     public void testContainer() {
```
Comment (diecui1202) [inline dubbo-container/dubbo-container-log4j/src/test/java/org/apache/dubbo/container/log4j/Log4jContainerTest.java]: Be aware of the code format. Besides, no assert statement in this case.

File: dubbo-container/dubbo-container-spring/src/test/java/org/apache/dubbo/container/spring/MainLauncherTest.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        new Thread(new Runnable() {			
+			@Override
+			public void run() {
+				try {
+					TimeUnit.SECONDS.sleep(5);
+				} catch (InterruptedException e) {
+				}
+				Class<?> obj = SpringContainer.getContext().getBean("container").getClass();
+		        Assert.assertEquals(SpringContainer.class,obj );
+		        System.exit(0);
```
Comment (diecui1202) [inline dubbo-container/dubbo-container-spring/src/test/java/org/apache/dubbo/container/spring/MainLauncherTest.java]: Why use System.exit() here? Code format too.

**Comment by diecui1202** [comment]
@stoneapple I'll close this PR, if necessary pls. send a new PR with the review comments. Thanks.

**Comment by lovepoem** [comment]
https://github.com/apache/incubator-dubbo/issues/1685

**Comment by lovepoem** [comment]
@stoneapple ,please see the review result

**Pull request closed without merging**